core/templates/components/question-directives/question-player/question-player-concept-card-modal.controller.spec.ts

```diff
@@ -129,7 +131,7 @@ describe('Question Player Concept Card Modal Controller', function() {
     expect($scope.modalHeader).toBe(undefined);
   });
 
-  it('should retry test', function() {
+  it('should refresh page when retrying a test', function() {
```

Comment:
... when the user retries a practice session

(I think? I am not really sure what the user is retrying in this context. /cc @aks681)

(Also, this is just to try and avoid confusion with the meaning of "test", which could possibly refer to frontend tests in this context.)

Reply:
I agree with you but I feel confused in some cases, like this one. I don't know either the test the user is trying but what I use to in these cases is to check what the function is doing but this one is not providing any useful information, it's just changing the window location, so I think it would help to have more jsdocs or even better function names. What do you think?

Reply:
Sure, PTAL @aks681

Reply:
The 'retry test' refers to the practice test that the learner was going through. This is called at the end of a practice session, if the learner wants to retry the test again, after having read the concept card for  some skills.

Reply:
Done.